channels/ae.m3u

```diff
@@ -233,8 +159,6 @@ https://weyyak-live.akamaized.net/weyyak_drama/index.m3u8
 https://weyyak-live.akamaized.net/weyyak_mix/index.m3u8
 #EXTINF:-1 tvg-id="WeyyakNawaem.ae" tvg-name="Weyyak Nawaem" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://content.weyyak.com/d867c2e9-328d-ea11-831d-02d662556796/poster-image" group-title="",Weyyak Nawaem (720p)
 https://weyyak-live.akamaized.net/weyyak_nawaem/index.m3u8
-#EXTINF:-1 tvg-id="Yas.ae" tvg-name="Yas" tvg-country="AE" tvg-language="" tvg-logo="" group-title="",Yas (1080p)
-https://admdn1.cdn.mangomolo.com/yastv/smil:yastv.stream.smil/playlist.m3u8?adtv
```

Comment:
Why, it works

Reply:
Duplicate of another one.

Reply:
> 
> 
> Why? that works. Overall, I see some channels being removed without a replacement. That's not our way to manage things

They were basically the same channels, but only a different chunklist. All of them work, but we keep only one adaptive link that simply do the same thing as keeping 5 or 6 chunk links.

Reply:
I see only one Yas here and it shows Yas icon on the channel

Reply:
Line 163.

Reply:
Also, we should retain all logos, groups etc if at least a single entry had them

Reply:
That's what I try to do.